erpnext/controllers/selling_controller.py

```diff
@@ -153,6 +153,10 @@ def calculate_contribution(self):
 				self.base_net_total * sales_person.allocated_percentage / 100.0,
 				self.precision("allocated_amount", sales_person))
 
+			sales_person.incentives = flt(
```

Comment:
I think commission_rate is not mandatory. So, if commission_rate is there then only it should calculate incentives. Otherwise, if the user enters incentives manually it will become zero.